trapping-rainwater.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Write a program that computes how much watera a elevation map is able to trap after raining."
+source: "Trapping Rain Water"
+source_url: "https://leetcode.com/problems/trapping-rain-water"
```

Comment:
Oh, you are absolutely right. I had not thought about permissions and rules. It'a a pity! I liked this exercise.